R/LearnerClust.R

```diff
@@ -32,6 +32,10 @@
 LearnerClust = R6Class("LearnerClust",
   inherit = Learner,
   public = list(
+    #' @field assignments (`NULL` | `integer(1)`)\cr
```

Comment:
I guess `integer(1)` is wrong; this is a vector, right?

Reply:
Also, we should have an autotest to check this field

Reply:
> I guess `integer(1)` is wrong; this is a vector, right?

Yep, I fixed that.

Reply:
> Also, we should have an autotest to check this field

I added a unit test that checks if `assignments` field exists when saving is on, and vice versa.